### PR TITLE
Fixed Dockerfile to be able to use beam or spark inside container

### PR DESCRIPTION
**Pull request by ricardocarvalhods**
## Description
Fixed being able to use beam or spark inside container.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

File: contributing/Dockerfile [hunk 1]
```diff
@@ -16,4 +16,23 @@ WORKDIR /code
...
\ No newline at end of file
+RUN pip install -e .
+
+# Install process tools
+RUN apt-get update && apt-get -y install procps
+RUN apt-get -y install software-properties-common
+RUN apt-add-repository 'deb http://security.debian.org/debian-security stretch/updates main'
+
+# Install OpenJDK-8
+RUN apt-get update && \
```
Comment (dvadym) [inline contributing/Dockerfile:27]: Why do we need JDK? Could you please add a comment about that?

**Review by dvadym — COMMENTED** [review]
Thanks! It looks great, I left a small question.

File: contributing/Dockerfile [hunk 2]
```diff
@@ -1,4 +1,4 @@
-FROM python:3.9.16-slim
+FROM python:3.9.16-slim-bullseye
```
Comment (chinmayshah99) [inline contributing/Dockerfile:1]: any particular advantage in using `bullseye` version?

**Review by chinmayshah99 — COMMENTED** [review]
LGTM, other than this  small clarification

File: contributing/Dockerfile (hunk 2 above)
Comment (ricardocarvalhods) [reply contributing/Dockerfile:1]: No reason. Only with `-slim` it was already `bullseye`, I just made it explicit as I'm using openjdk's image for `bullseye` as well.

**Review by dvadym — APPROVED** [review]
Thanks!

**Pull request merged**